lib/raygun.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const env = require('env-var');
+
+module.exports = require('rhmap-raygun-nodejs')({
+  apiKey:  env('RAYGUN_API_KEY', 'key').asString()
```

Comment:
You don't need the apiKey now. Just pass in `{}`. You have to define the options parameter, but everything inside it is optional. The api key [defaults](https://github.com/TkeITMobility/rhmap-raygun-nodejs/blob/master/index.js#L15) to env('RAYGUN_API_KEY').

Also, we should probably update the readme to mention that you NEED to have this env var set.